### ISSUE TITLE
Change command to open new terminal

### ISSUE THREAD

**Issue by svzosorio** [jeremyramin/terminal-plus#251]
Actually the `cmd + shift + t` its overryde the command to open last closed tab.

**Comment by bmcallis** [comment]
I agree, re-opening a closed tab is something I do all the time. I don't frequently open new terminals though which made this extremely annoying to me. To get around it I updated my keymap.cson to unset the new terminal shortcut with this.

```
'.platform-darwin atom-workspace':
  'shift-cmd-T': 'unset!'
```
